src/series.rs

```diff
@@ -87,18 +87,33 @@ pub enum BarType {
 
 #[derive(Properties, Clone)]
 pub struct Props {
+    /// A vector of data points that represents the series, along with optional labels at each point
     pub data: Rc<Data>,
+    /// The SVG height of the series
     pub height: f32,
+    /// The scaling factor for data along the x axis
     pub horizontal_scale: Rc<dyn Scale>,
+    /// The horizontal scale step is used to determine when there is a gap in data, such that
+    /// if a line chart was drawn, then if two data items are separated by more than this can,
+    /// the line will end and start again. For scatter plots, this property does not get used. I shall document it.
```

Comment:
`I shall document it.`

Perhaps write this attribute with a `prop_or_default` flag so that it does not need to be specified in scatter plots

Reply:
Good idea

Reply:
I made it an option so that I can provide a default - same, same